lib/app_routes/app_routes.dart

```diff
@@ -1,22 +1,26 @@
+import 'package:book/presentation/book/add_new_book_page.dart';
 import 'package:book/presentation/book/view/home_page_view.dart';
 import 'package:book/presentation/login/view/login_page_view.dart';
 import 'package:book/presentation/register/view/register_page_view.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter/widgets.dart';
 
-const String homeRoute = '/';
-const String loginRoute = '/Login';
-const String registerRoute = '/Register';
+const String kHomeRoute = '/';
+const String kLoginRoute = '/Login';
+const String kRegisterRoute = '/Register';
+const String kAddNewBookPage = '/AddNewBookPage';
```

Comment:
make better name, it look's like adding book's page, and all other constants are with Route at the end, but this one is with Page